apps/microtvm/zephyr/template_project/src/host_driven/semihost.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
can you move fvp related files to a sub-directory like: `apps/microtvm/zephyr/template_project/src/host_driven/fvp`
it would make it more readable. 
Also using this style, we could also separate the zephyr transport files from main.c and move it to a different sub directory . That could be for a separate PR.

Reply:
done